### PR TITLE
Add endpoint to get version info

### LINKED ISSUE #2234: Create a new endpoint to show the current app version

### PR DESCRIPTION
**Pull request by sainak**
## Proposed Changes

- show the app version from the env

### Associated Issue

- closes #2234

### Architecture changes

- Remove this section if not used

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [ ] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

File: config/settings/deployment.py [hunk 1]
```diff
@@ -7,13 +7,13 @@
 from sentry_sdk.integrations.redis import RedisIntegration
 
 from .base import *  # noqa
-from .base import env
+from .base import APP_VERSION, DATABASES, TEMPLATES, env
```
Comment (vigneshhari) [inline config/settings/deployment.py:10]: You need to import * from base right ? the configs in base are all needed in deployment.

**Review by vigneshhari — CHANGES_REQUESTED** [review]

File: config/settings/deployment.py (hunk 1 above)
Comment (sainak) [reply config/settings/deployment.py:10]: yes
the second import is to avoid having to ignore possibly undefined lint checks

Comment (sainak) [reply config/settings/deployment.py:10]: it will still import rest of the config fron base

Comment (vigneshhari) [reply config/settings/deployment.py:10]: nvm, my eyes aint working properly 😅

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**